lib/graphql/schema/printer.rb

```diff
@@ -44,6 +48,10 @@ def print_schema_definition(schema)
       BUILTIN_SCALARS = Set.new(["String", "Boolean", "Int", "Float", "ID"])
       private_constant :BUILTIN_SCALARS
 
+      def is_spec_directive(directive)
+        ['skip', 'include', 'deprecated'].include?(directive.name)
+      end
```

Comment:
You're right, maybe I don't understand! If graphql-ruby supports `@stream` and `@defer`, but they aren't part of the GraphQL spec, would they show up in a schema print-out or not?